packages/vscode-extension/src/debug/teamsfxDebugProvider.ts

```diff
@@ -64,7 +64,17 @@ export class TeamsfxDebugProvider implements vscode.DebugConfigurationProvider {
           debugConfiguration.timeout = 20000;
         }
 
-        const debugConfig = await commonUtils.getDebugConfig(isLocalSideloadingConfiguration);
+        let debugConfig = undefined;
+        if (isLocalSideloadingConfiguration) {
```

Comment:
What if `isLocalSideloadingConfiguration` is false?

Reply:
My bad, I have updated this part.